user.py

```diff
@@ -0,0 +1,44 @@
+import pickle
+import uuid
+
+
+class User:
+
+        def __init__(self, fullname, screenname):
+            self.screen_name = screenname
+            self.full_name = fullname
+            self.unique_user_ID = uuid.uuid4().int
+            self.user_object = {"uuid": self.unique_user_ID, "full name": self.full_name, "screen name": self.screen_name}
+            print(self.unique_user_ID)
+            self.serialize_user()
+
+        def serialize_user(self):
```

Comment:
> **Pro tip:** You can just call this method `serialize`.  Adding the `_user` after it is redundant since it is a method of a user.  Same with the deserialize method below.